core/admin/mailu/models.py

```diff
@@ -281,9 +281,14 @@ def dns_autoconfig(self):
     def dns_tlsa(self):
         """ return TLSA record for domain when using letsencrypt """
         hostname = app.config['HOSTNAME']
-        if app.config['TLS_FLAVOR'] in ('letsencrypt', 'mail-letsencrypt'):
-            # current ISRG Root X1 (RSA 4096, O = Internet Security Research Group, CN = ISRG Root X1) @20210902
-            return f'_25._tcp.{hostname}. 86400 IN TLSA 2 1 1 0b9fa5a59eed715c26c1020c711b4f6ec42d58b0015e14337a39dad301c5afc3'
+        if True:# app.config['TLS_FLAVOR'] in ('letsencrypt', 'mail-letsencrypt'):
```

Comment:
Isn't it possible to use a paid certificate qith 1 year lifetime for web and mail in the non-letsencrypt flavors?
Why should we show the lets encrypt TLSA entries in this case?
I don't think it would be possible to show the "correct" TLSA records for people using their own certificates, bat maybe there can be anotice shown in the webinterface.
This change is - if we take it serieously - an API change (string -> list of strings).
The config export and the REST API are both an API.
Shouldn't the REST API be changed to `fields.List(fields.String(...))`?

Reply:
I agree, the API should be revisited but that stands true for other fields too (DKIM...). I've made the minimal change in 58b1738d and fixed the non LE case in 0171c6d.

Regarding the display of TLSA records for non LE certs, I don't think it would help. We don't want to be in the business of advising people what their TLSA records should be; We would need to define whether we're encouraging them to pin the end-entity certificate (``3 1 1`` record) or the CA like we do (``2 1 1``) .

I have made a comment on #3113 regarding re-visiting the API

Reply:
I didn't want to suggest we should display TLSA strings for non-letsencrypt setups.
This is why I think the `TLS_FLAVOR` still should be prerequisite for displaying the TLSA records.